package.json

```diff
@@ -33,7 +33,7 @@
   "devDependencies": {
     "@commitlint/cli": "8.2.0",
     "@commitlint/config-angular": "8.2.0",
-    "@edx/frontend-build": "2.0.6",
+    "@edx/frontend-build": "^5.3.2",
```

Comment:
Should we pin this dependency so we're consistently pinning all dependencies?

Reply:
Fixed!